tests/check_consistent.py

```diff
@@ -97,22 +98,21 @@ def check_same_files():
                 )
 
 
+_VERSIONS_RE = re.compile(r"^([a-zA-Z_][a-zA-Z0-9_]*): ([23]\.\d{1,2})(-([23]\.\d{1,2}))?$")
```

Comment:
Should this support (or even require) `foo: x.y-` syntax instead of `foo: x.y` when a module `foo` is new in Python x.y?

Reply:
I'll just add `-` to all versions and check that here.